### DISCUSSION TITLE
Update PFX certificate

### DISCUSSION THREAD

**Discussion by EdersonRavizza4Matt** [requarks/wiki#6522] [Help / Questions] [answered]
Hello, I just got the administration of the wiki.js environment in the company where I work.

It turns out that the digital certificate is about to expire tomorrow and a new one was generated to replace it, but I couldn't find how to change the certificate in the admin session, could you help me?

Where should I insert the new certificate or what is another way to do this procedure?

Thanks.

Ederson Ravizza.

**Comment by P4mes4n** [comment, marked as answer]
There is no way to change the certificate via the GUI. You have to manually place the certificate in the folder that is specified in the `config.yml` of your Wiki.

https://docs.requarks.io/install/config#https
